homeassistant/components/system_bridge/sensor.py

```diff
@@ -252,6 +285,275 @@ def extra_state_attributes(self) -> dict[str, Any]:
         }
 
 
+class SystemBridgeGpuMemoryFreeSensor(SystemBridgeSensor):
+    """Defines a GPU memory free sensor."""
+
+    def __init__(
+        self, coordinator: SystemBridgeDataUpdateCoordinator, index: int, name: str
+    ) -> None:
+        """Initialize System Bridge sensor."""
+        super().__init__(
+            coordinator,
+            f"gpu_{index}_memory_free",
+            f"{name} Memory Free",
+            "mdi:memory",
+            None,
+            DATA_GIGABYTES,
+            True,
```

Comment:
It would make it a lot more readable if we used EntityDescription for these things. We could define a formatting lambda that we store in the description. We would probably be able to drop the subclasses here then. Eg:

https://github.com/home-assistant/core/blob/4903c1fbfd6b7ebcaa6b6a5c202a35939bac0de6/homeassistant/components/airly/const.py#L81-L88